vta/src/verilator/dpi_module.cc

```diff
@@ -0,0 +1,353 @@
+#include <tvm/runtime/module.h>
```

Comment:
For all files, add ASF license header as in https://github.com/dmlc/tvm/blob/master/vta/include/vta/runtime.h#L2